### PR TITLE
Completed html-web-page - YoonHwan Cho

### PR DESCRIPTION
**Pull request by YoonHCho**
Completed html-web-page - YoonHwan Cho
HTML Challenge

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        <td>$30</td>
+        <td>$50</td>
+        <td>$60</td>
+      </tr>
+    </tfoot>
+  </table>
+  <p>Like what you see? Become a part of the JOTMC fmaily today!</p>
+  <form>
+    <div>
+      <label>
```
Comment (codingcodymiller) [inline html-web-page/index.html:76]: There is no text labeling this `input`, so no `label` element is necessary

File: html-web-page/index.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        <input type="text" name="userName" placeholder="Your Name">
+      </label>
+      <label>
```
Comment (codingcodymiller) [inline html-web-page/index.html:79]: There is no text labeling this `select`, so no `label` element is necessary

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by YoonHCho** [comment]
removed label from both `input` and `select`.
so, if there is no actual text input from user, there is no need to wrapt around label?

**Comment by codingcodymiller** [comment]
> removed label from both `input` and `select`. so, if there is no actual text input from user, there is no need to wrapt around label?

That is correct!

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**